gtests/net/mptcp/add_addr/add_addr_retry_plain.pkt

```diff
@@ -19,8 +19,8 @@
 +0       >   .  1:1(0)        ack 1           <add_address address_id=1  addr[saddr0] hmac=auto>
 
 // Retransmitted after one second
-+1.0     >   .  1:1(0)        ack 1           <add_address address_id=1  addr[saddr0] hmac=auto>
-+1.0     >   .  1:1(0)        ack 1           <add_address address_id=1  addr[saddr0] hmac=auto>
++0~+1.0  >   .  1:1(0)        ack 1           <add_address address_id=1  addr[saddr0] hmac=auto>
++0~+1.0  >   .  1:1(0)        ack 1           <add_address address_id=1  addr[saddr0] hmac=auto>
```

Comment:
I understand that this will make the tests passing, but we will no longer validate the precision of the timeout... If we cannot "precisely" predict the retransmission timeout at that this stage, we could do what you did, and only focus on the number of retransmissions. But if in your observation, the RTO is usually around 0.65s, I would suggest setting `+0.65` instead of a "large" range.

Also, 0.65s looks a lot to me: The 3rd ACK at line 15 should be injected with a delay of 0.01s.

What are the values of `icsk->icsk_rto`, `tp->srtt_us`, `tp->rttvar_us`, `__tcp_set_rto(tp)` and the one you set for the timer when scheduling each retransmissions and when running this test?

(for the rto value in jiffies, please convert them to usec like the others → `jiffies_to_usecs()`)